src/Microsoft.Fast.Components.FluentUI/Components/Slider/FluentSlider.razor.cs

```diff
@@ -60,7 +60,7 @@ protected override bool TryParseValueFromString(string? value, [MaybeNullWhen(fa
         }
         else
         {
-            validationErrorMessage = string.Format(CultureInfo.InvariantCulture, "The {0} field must be a number.", DisplayName ?? FieldIdentifier.FieldName);
+            validationErrorMessage = string.Format(CultureInfo.InvariantCulture, "The {0} field must be a number.", DisplayName ?? (FieldBound ? FieldIdentifier.FieldName : "(unknown)"));
```

Comment:
Since you use `(unknown)`, could you add this value in an internal constant (FluentInputBase)?

Reply:
Done!